source/en/api/api_clustering.rst

```diff
@@ -165,7 +217,13 @@ Methods
 
       :return:     coreset of cluster
 
-      Returns coreset of cluster
+      Returns coreset of cluster in datum
+
+   .. mpidl:method:: list<list<weighted_index > > get_core_members_light()
```

Comment:
Please add description that dbscan does not support get_k_centers, get_nearest_center, get_nearest_members_light